sound/soc/sof/topology.c

```diff
@@ -1892,6 +1903,13 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 		}
 	}
 
+	/* TODO: free sroute/connect when unloading route */
```

Comment:
well, duh. Can you please update this one, this will create issues with module load/unload and break CI.

Reply:
@plbossart sure, I am working on it. It wasnt very straightforward which route is being unloaded from the arguments. I need to study this a bit more to implement it correctly.

Reply:
You will need to deep dive this more as I'm short of time and may also need to modify the API if needed to pass in the extra info.